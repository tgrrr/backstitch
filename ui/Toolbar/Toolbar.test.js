import React from 'react'
import renderer from 'react-test-renderer'
import { Toolbar } from '.'

it('Toolbar: default', () => {
    const component = renderer.create(
        <Toolbar>
            <>
                Toolbar
            </>
        </Toolbar>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
