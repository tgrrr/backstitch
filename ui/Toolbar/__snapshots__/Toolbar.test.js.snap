// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toolbar: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Toolbar "
    data-testid="Toolbar"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      Toolbar
    </div>
  </div>
</div>
`;
